conda_smithy/feedstock_tokens.py

```diff
@@ -413,17 +421,29 @@ def register_feedstock_token(
                 json.dump(token_data, fp)
 
             # push
-            repo.index.add(token_file)
-            repo.index.commit(
-                "[ci skip] [skip ci] [cf admin skip] ***NO_CI*** "
-                "added token for {}/{} on provider{}".format(
-                    user,
-                    project,
-                    "" if provider is None else " " + provider,
-                )
+            repo.index.add(os.path.relpath(token_file, tmpdir))
+            repo.index.write()
+            subprocess.run(
+                [
+                    "git",
+                    "commit",
+                    "-q",
+                    "-m",
+                    "[ci skip] [skip ci] [cf admin skip] ***NO_CI*** "
+                    "added token for {}/{} on provider{}".format(
+                        user,
+                        project,
+                        "" if provider is None else " " + provider,
```

Comment:
Since we are touching this string anyway, what about some f-string niceness?

Reply:
In fact, it was itching me to change it, so I'll do that as well.